probe-rs-cli-util/src/common_options.rs

```diff
@@ -508,3 +492,11 @@ pub fn print_families(mut f: impl Write) -> Result<(), OperationError> {
     }
     Ok(())
 }
+
+pub fn common_arguments() -> Vec<String> {
+    FlashOptions::ARGUMENTS
+        .iter()
+        .chain(ProbeOptions::ARGUMENTS.iter())
+        .map(|arg| arg.to_string())
+        .collect()
+}
```

Comment:
```suggestion
pub fn common_arguments() -> Vec<&'static str> {
    FlashOptions::ARGUMENTS
        .iter()
        .chain(ProbeOptions::ARGUMENTS)
        .copied()
        .collect()
}
```

You can avoid the `to_string()` by using `copied()`, this will convert `&&str` back into just `&str`.